quipucords/scanner/network/inspect.py

```diff
@@ -122,6 +122,7 @@ def inspect_scan(self):
             'etc_release',
             'file_contents',
             'jboss_eap',
+            'jboss_brms',
```

Comment:
I intentionally went out of alphabetical order here because there are some optional BRMS checks that rely on EAP already having been scanned.